.env.example

```
@@ -24,5 +24,14 @@ RESEND_FROM_EMAIL=noreply@yourdomain.com
 RESEND_FROM_NAME=YourAppName
 RESEND_AUDIENCE_ID=<your-resend-audience-id>
 
+# PostHog (Analytics)
+POSTHOG_ACTIVATED=true
+NEXT_PUBLIC_POSTHOG_ACTIVATED=true
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_POSTHOG_ACTIVATED key should go before the POSTHOG_ACTIVATED key

(UnorderedKey)